styles/blocks/_buttons.scss

```diff
@@ -1,12 +1,3 @@
-a.action.primary,
-button {
-    border-radius: 3px;
-}
-button {
-    &:active {
-        box-shadow: $button__shadow;
-    }
-}
```

Comment:
Why is this code removed? It look like in the core - https://github.com/magento/magento2/blob/develop/app/design/frontend/Magento/blank/web/css/source/_buttons.less